CMakeLists.txt

```diff
@@ -153,6 +153,13 @@ if(PACKAGE_TESTS)
         add_subdirectory(${PROJECT_SOURCE_DIR}/test)
     endif(CMAKE_VERSION VERSION_LESS 3.11)
 endif()
+#########################################################################
+################# BENCHMARKING ##########################################
+option(PACKAGE_BENCHMARKS "Build the benchmarks" ON)
```

Comment:
I am thinking that maybe this should be OFF by default?
This is more like an internal developer thing rather than something needed by the user.

Reply:
Is now off by default.